test/components/converters/test_pypdf_to_document.py

```diff
@@ -113,8 +113,8 @@ def test_default_convert(self):
             layout_mode_font_height_weight=1.5,
         )
 
-        doc = converter._default_convert(mock_reader)
-        assert doc.content == "Page 1 content\fPage 2 content"
+        text = converter._default_convert(mock_reader)
```

Comment:
testing against private methods seems not really best practice but I guess that's a thing for a different time 😁